CHANGE_LOG.md

```diff
@@ -11,7 +11,8 @@ All notable changes to this project will be documented in this file.
 ### Added
 
 - Added new middleware (messageSequenceIdOverrideMiddleware) to ensure proper order of messages
-- Added handling for OriginalMessageId in transcript component to fix sorting issues.
+- Added handling for OriginalMessageId in transcript component to fix sorting issues
```

Comment:
The changes are under chat-component, not chat-widget

Reply:
fixed